### PR TITLE
Create database from diagram

### PR DESCRIPTION
**Pull request by nicupop729**
## This feature branch holds the following:

- Create a file named schema_based_on_diagram.sql where we implement the database from the diagram;
- Add FOREIGN KEYs indexes;
- Join tables from many-to-many relationships;

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+    CONSTRAINT treatment_fk
+    FOREIGN KEY (treatment_id) REFERENCES treatments (id)
+);
+
+SELECT * FROM medical_histories
+INNER JOIN medical_history_treatments
+ON medical_histories.id = medical_history_treatments.medical_history_id
+INNER JOIN treatments
+ON medical_history_treatments.treatments_id = treatments.id;
```
Comment (rukundoeric) [inline schema_based_on_diagram.sql]: - [ ] <b>REQUIRED:</b> You haven't created all indexes the project requires you to create for all FOREIGN KEYs. Kindly do that.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @nicupop729, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (nicupop729) [reply schema_based_on_diagram.sql]: Hello @rukundoeric,
First of all, thanks a lot for the comment and for the review.
But to understand well what more is required from us,  please bring more clarification.
We created all the tables from the diagram.
After that, we linked them together (altering the tables) thru the foreign keys, as the diagram shows.
And at the end, in the case of many to many relations between medical_histories and treatments tables, we created an extra table.

Thanks a lot for the clarification.

Comment (nicupop729) [reply schema_based_on_diagram.sql]: This is the diagram that our code produces - created on [dbdiagram.io](https://dbdiagram.io/d/61e6e6e6bb7a646986f8b1aa)

diagram

**Review by rukundoeric — APPROVED** [review]
Hi @nicupop729,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**